src/game/default/g_combat.c

```diff
@@ -137,21 +137,28 @@ static int16_t G_CheckArmor(g_entity_t *ent, const vec3_t pos, const vec3_t norm
 	const int16_t quantity = ent->client->locals.inventory[ITEM_INDEX(armor)];
 
 	vec_t protection = 0.0;
+	vec_t burnrate = 1.0;
 
-	switch (armor->tag) {
+	// energy weaps cut through armor more
+	const int16_t type = (dflags & DMG_ENERGY) ? armor->tag-1 : armor->tag;
+
+	switch (type) {
 		case ARMOR_BODY:
-			protection = 1.0;
+			protection = 0.75;
+			burnrate = 1.35;
 			break;
 		case ARMOR_COMBAT:
 			protection = 0.66;
+			burnrate = 1.25;
 			break;
 		case ARMOR_JACKET:
-			protection = 0.5;
+			protection = 0.33;
+			burnrate = 1.15;
 			break;
 	}
 
 	const int16_t saved = Clamp(protection * damage, 0, quantity);
-	ent->client->locals.inventory[ITEM_INDEX(armor)] -= saved;
+	ent->client->locals.inventory[ITEM_INDEX(armor)] -= (saved / burnrate);
```

Comment:
`saved * burnrate` would yield > 1 unit of armor for every point of damage saved, which is what we want, right? I guess I should look at Q2 / Q3 to see how they expend armor..